x-pack/plugins/security_solution/public/onboarding/components/onboarding_body/hooks/use_completed_cards.ts

```diff
@@ -109,17 +118,35 @@ export const useCompletedCards = (cardsGroupConfig: OnboardingGroupConfig[]) =>
     [setCardComplete, setCardCheckCompleteResult]
   );
 
+  const displayErrorToast = useCallback(
+    (cardTitle: string, error: Error) => {
+      toasts.addError(error, {
+        title: getErrorMessage(cardTitle),
+      });
+    },
+    [toasts]
+  );
+
   const checkCardComplete = useCallback(
     (cardId: OnboardingCardId) => {
       const cardConfig = cardsWithAutoCheck.find(({ id }) => id === cardId);
 
       if (cardConfig) {
-        cardConfig.checkComplete?.(services).then((checkCompleteResult) => {
-          processCardCheckCompleteResult(cardId, checkCompleteResult);
-        });
+        cardConfig
+          .checkComplete?.(services)
+          .catch((e: Error) => {
+            displayErrorToast(cardConfig.title, e);
```

Comment:
We already have the `services` here, no need to add another dependency, we can do:
```suggestion
           services.notifications.toasts.addError(err, { title: cardConfig.title });
```